openid-connect-server-webapp/src/main/webapp/WEB-INF/views/logout.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="o" tagdir="/WEB-INF/tags"%>
+<%@ taglib prefix="security" uri="http://www.springframework.org/security/tags"%>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
+
+<spring:message code="logout.title" var="title"/>
+<o:header title="${title}"/>
+<o:topbar pageName="Logout"/>
+<div class="container-fluid main">
+	<div class="row-fluid">
+		<div class="span1"></div>
+		<div class="span10">
+            <security:authorize access="hasRole('ROLE_USER')">
+				<div class="hero-unit">
+					<h2><spring:message code="logout.title"/></h2>
+					<p>
+						<spring:message code="logout.body"/>
+					</p>
+					<p>
+						<a href="" class="btn btn-primary btn-large endSessionLogoutLink"><spring:message code="topbar.logout"/></a>
```

Comment:
In the case of the linked logout page, it's because there are 2 "logoutLinks" (only one shown at a time depending on viewport size, relying on Bootstrap "responsiveness"), and instead of duplicating the form (and using submit buttons), the author chose to instead rely on JavaScript.